modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/test/java/com/liferay/dynamic/data/mapping/model/impl/DDMFieldAttributeImplTest.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.dynamic.data.mapping.model.impl;
+
+import com.liferay.dynamic.data.mapping.model.DDMFieldAttribute;
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.test.rule.LiferayUnitTestRule;
+
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+/**
+ * @author Carolina Barbosa
+ */
+public class DDMFieldAttributeImplTest {
+
+	@ClassRule
+	@Rule
+	public static final LiferayUnitTestRule liferayUnitTestRule =
+		LiferayUnitTestRule.INSTANCE;
+
+	@Test
+	public void testSetAttributeValue() {
+		DDMFieldAttribute ddmFieldAttribute = new DDMFieldAttributeImpl();
+
+		ddmFieldAttribute.setAttributeValue(_repeat("a", 150));
+
+		Assert.assertEquals(
+			StringPool.BLANK, ddmFieldAttribute.getLargeAttributeValue());
+		Assert.assertEquals(
+			ddmFieldAttribute.getAttributeValue(),
+			ddmFieldAttribute.getSmallAttributeValue());
+
+		ddmFieldAttribute.setAttributeValue(_repeat("á", 150));
```

Comment:
here too? :thinking:

Reply:
but here I specifically need multibyte characters (like accented characters) to exceed the maximum size of 255.

`á = 2` bytes so it will result on 300 bytes

Reply:
I guess we can try this way :pray: :grimacing: